test_app/after/src/others.erl

```diff
@@ -44,6 +44,27 @@ receive_expr() ->
     receive
         with ->
             {no, timeout}
+    end,
+    receive after 1000 ->
+        {only, timeout}
+    end,
+    receive
+        with ->
+            expression and also
+    after 1000 ->
+        timeout
+    end,
+    receive after with:a(very,
+                         long and strange,
+                         but,
+                         possible,
+                         timeout,
+                         expression,
+                         that,
+                         is,
+                         super,
+                         extensive) ->
+        yeah
```

Comment:
Same comment as above…
```erlang
receive after …
```
…or…
```erlang
receive
after …
```
?

Reply:
If you're going for "saving space" you should align it to the left, as much as possible, but I'd be contradicting myself if I said something different here.